build/linux/installer/scripts/dcr-config-parser.rb

```diff
@@ -33,12 +33,12 @@
     end
 
     # Raise an error if no JSON file is found
-    raise 'No JSON file found in the specified directory' unless file_path
+    raise 'No JSON file found in the specified directory. Check if mdsd is running in MSI mode' unless file_path
```

Comment:
If the Geneva Integration enabled, we shouldnt get into this error since there is no DCR. Do you know why this error triggered  when the geneva integration enabled?

Reply:
yes, we check whether geneva integration is enabled and return if it is: https://github.com/microsoft/Docker-Provider/blob/05e094df2470e8a6dcedd73b220c4fd8b7b27131/build/linux/installer/scripts/dcr-config-parser.rb#L18

it looks like for some reason cluster was in geneva mode but GENEVA_LOGS_INTEGRATION was not set

Reply:
Ok, lets investigate and also, we should also add similar check on GENEVA_LOGS_INTEGRATION_SERVICE_MODE as well. return if ENV['GENEVA_LOGS_INTEGRATION_SERVICE_MODE ']&.strip&.casecmp?('true')

Reply:
@pfrcks , do you know why GENEVA_LOGS_INTEGRATION flag not being set even though geneva enabled? can you please check Nont cluster and see what's the issue here?